composer.json

```diff
@@ -90,7 +90,7 @@
         "spryker/agent-auth-rest-api": "^1.0.0",
         "spryker/alternative-products-rest-api": "^1.1.2",
         "spryker/app-catalog-gui": "^1.2.0",
-        "spryker/asset": "^1.5.1",
+        "spryker/asset": "^1.4.0",
```

Comment:
Why?

Reply:
Looks like a mistake during the merging internal branch